src/main/groovy/org/jenkinsci/plugins/plumber/model/Reporter.groovy

```diff
@@ -46,6 +46,17 @@ public class Reporter extends AbstractPlumberModel {
 
     }
 
+    public Reporter(Map<String,Object> args) {
+        if (args != null) {
+            validateMapFields(args.keySet())
+
+            this.name = args.name
+            if (args.containsKey("config") && args.config instanceof Map) {
+                this.config = new MappedClosure((Map<String, Object>) args.config)
```

Comment:
@abayer why do you using `this` everywhere? Ideally you shouldn't use overlapped variable names at all.